docs/en/gestione_errori.rst

```diff
@@ -0,0 +1,54 @@
+.. include:: ../common/common_definitions.rst
+
+.. _gestione_errori: 
+
+Error Management
+================
+
+In case of errors, the OP displays error messages about the OpenID Connect interchanges, 
```

Comment:
I never seen the term interchanges in IETF/OIDF specs

may we have to rephrase it?

Reply:
https://github.com/italia/spid-cie-oidc-docs/pull/51/commits/fd19ce20aa4e4fe864eda359f53f3c2c19759302